lib/test/tester.js

```diff
@@ -259,26 +259,31 @@ class Tester {
   }
 
   createCoverageFile() {
-    this.removeTempDir()
-    if (!this.tempDir) {
-      this.tempDir = fs.realpathSync(temp.mkdirSync())
+    const additionalTestArgs = atom.config.get('go-plus.config.additionalTestArgs')
+    const res = additionalTestArgs.match(/-coverprofile=([A-Za-z0-9/\._-]+)/);
```

Comment:
This regex won't match windows paths because they use `\` instead of `/`.

Reply:
Changed to `[^ ]+`.

Reply:
Thas was a bad idea. It's now `[^\s]`.